src/EtwClrProfiler/ETWClrProfilerX86.vcxproj

```diff
@@ -66,6 +66,7 @@
       <ExceptionHandling>false</ExceptionHandling>
       <BufferSecurityCheck>false</BufferSecurityCheck>
       <PreprocessToFile>false</PreprocessToFile>
+      <AdditionalIncludeDirectories>$(NetFxKitsDir)\include\um</AdditionalIncludeDirectories>
```

Comment:
❔ Is there a reason why this one doesn't inherit previous values (i.e. `%(AdditionalIncludeDirectories)`)?